### PR TITLE
Fixes #111

### LINKED ISSUE #111: List known third party packages and document suggested package naming convention
Mentioned in #110

### PR DESCRIPTION
**Pull request by frekyll**
I added a section in the main docs to address issue #111.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,6 +187,23 @@ Tools and dependency managers:
...
 
+### Community Packages
+
+Easily extend SUIT CSS with these third-party packages.
+
+#### Components
+
+* https://github.com/simonsmith/suitcss-components-form-field
+* https://github.com/giuseppeg/suitcss-components-icon
+* https://github.com/giuseppeg/suitcss-components-arrow
```
Comment (giuseppeg) [inline README.md]: mine repo are old can you replace the two links above with https://github.com/giuseppeg/suitcss-toolkit (not maintained anyway)

File: README.md [hunk 2]
```diff
@@ -187,6 +187,23 @@ Tools and dependency managers:
...
+* https://github.com/antontrollback/select
+
+#### Utilities
+
+* https://github.com/simonsmith/suitcss-utils-list
+* https://github.com/simonsmith/suitcss-utils-image
+* https://github.com/frekyll/suitcss-utils-spacing
+
 ## Development
```
Comment (giuseppeg) [inline README.md]: can you move this section up, before `Example` ?

Comment (frekyll) [reply README.md]: I made those changes.

File: README.md [hunk 3]
```diff
@@ -68,6 +68,22 @@ the `-w` flag e.g. `npm run build -- -w`.
 Refer to the [SUIT theme](https://github.com/suitcss/theme) for a more thorough
 example.
 
+## Community Packages
+
+Easily extend SUIT CSS with these third-party packages.
```
Comment (simonsmith) [inline README.md]: I think we could probably lose these sentence. Otherwise looks good!

Comment (frekyll) [reply README.md]: I removed line 73. Is that what you were referring to?

**Comment by simonsmith** [comment]
Looks great, thanks!

**Pull request merged**